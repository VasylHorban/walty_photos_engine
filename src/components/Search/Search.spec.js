// import Search from './Search';
// import ShallowMock from '../../ShallowMock'
// import { Provider } from 'react-redux';
// const mockStore = {
//     state : {
//         search : {
//             inputValue : 'test'
//         }
//     },
//     dispatch : () =>{}
// }
// const setUp = shallow(<Provider store = {mockStore}><Search/></Provider>);

// describe('Search component', () => {
//     let component;
//     let instanse;
//     beforeEach(()=> {
//         component = setUp;
//         instanse = component.instanse;
//     })
//   it('should render Search component', () => {
//     expect(component).toMatchSnapshot();
//   });
// });