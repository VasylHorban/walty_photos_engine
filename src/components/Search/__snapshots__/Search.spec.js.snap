// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component should render Search component 1`] = `
<Search
  search={
    Object {
      "inputValue": "test",
    }
  }
/>
`;
